examples/code/command-line-parsing/basic_md5_with_optional_file_broken/build_basic_md5_with_optional_file_broken.errsh

```diff
@@ -1,8 +1,8 @@
+  (cd _build/default && /home/yminsky/.opam/fresh-4.06.1/bin/ocamlc.opt -w -40 -g -bin-annot -I .basic_md5_with_optional_file_broken.eobjs -I /home/yminsky/.opam/fresh-4.06.1/lib/base -I /home/yminsky/.opam/fresh-4.06.1/lib/base/caml -I /home/yminsky/.opam/fresh-4.06.1/lib/base/md5 -I /home/yminsky/.opam/fresh-4.06.1/lib/base/shadow_stdlib -I /home/yminsky/.opam/fresh-4.06.1/lib/bin_prot -I /home/yminsky/.opam/fresh-4.06.1/lib/bin_prot/shape -I /home/yminsky/.opam/fresh-4.06.1/lib/core -I /home/yminsky/.opam/fresh-4.06.1/lib/core_kernel -I /home/yminsky/.opam/fresh-4.06.1/lib/core_kernel/base_for_tests -I /home/yminsky/.opam/fresh-4.06.1/lib/cryptokit -I /home/yminsky/.opam/fresh-4.06.1/lib/fieldslib -I /home/yminsky/.opam/fresh-4.06.1/lib/jane-street-headers -I /home/yminsky/.opam/fresh-4.06.1/lib/ocaml/threads -I /home/yminsky/.opam/fresh-4.06.1/lib/parsexp -I /home/yminsky/.opam/fresh-4.06.1/lib/ppx_assert/runtime-lib -I /home/yminsky/.opam/fresh-4.06.1/lib/ppx_bench/runtime-lib -I /home/yminsky/.opam/fresh-4.06.1/lib/ppx_compare/runtime-lib -I /home/yminsky/.opam/fresh-4.06.1/lib/ppx_expect/collector -I /home/yminsky/.opam/fresh-4.06.1/lib/ppx_expect/common -I /home/yminsky/.opam/fresh-4.06.1/lib/ppx_expect/config -I /home/yminsky/.opam/fresh-4.06.1/lib/ppx_hash/runtime-lib -I /home/yminsky/.opam/fresh-4.06.1/lib/ppx_inline_test/config -I /home/yminsky/.opam/fresh-4.06.1/lib/ppx_inline_test/runtime-lib -I /home/yminsky/.opam/fresh-4.06.1/lib/ppx_sexp_conv/runtime-lib -I /home/yminsky/.opam/fresh-4.06.1/lib/sexplib -I /home/yminsky/.opam/fresh-4.06.1/lib/sexplib/unix -I /home/yminsky/.opam/fresh-4.06.1/lib/sexplib0 -I /home/yminsky/.opam/fresh-4.06.1/lib/spawn -I /home/yminsky/.opam/fresh-4.06.1/lib/splittable_random -I /home/yminsky/.opam/fresh-4.06.1/lib/stdio -I /home/yminsky/.opam/fresh-4.06.1/lib/typerep -I /home/yminsky/.opam/fresh-4.06.1/lib/variantslib -I /home/yminsky/.opam/fresh-4.06.1/lib/zarith -no-alias-deps -o .basic_md5_with_optional_file_broken.eobjs/basic_md5_with_optional_file_broken.cmo -c -impl basic_md5_with_optional_file_broken.ml)
   File "basic_md5_with_optional_file_broken.ml", line 17, characters 4-11:
-  Error: This expression has type string -> unit -> unit
+  Error: This expression has type Base.string -> unit -> unit
```

Comment:
It's a pity short paths doesnt convert this Base.string to string

Reply:
When Leo's short-paths patch lands, this issue should be resolved.